app/controllers/api/v1/organization_controller.rb

```diff
@@ -48,7 +48,7 @@ def create
 
     result = @organizationRepository.create_organization({:name => name})
 
-    render stauts: 200,
+    render status: result[:status],
```

Comment:
This is great! But now that `render` expects the result to have a `status` propety, we need to make sure every return statement in the Repository.create_organization also includes a status. Or we should have a default status here